src/main/resources/db/migration/V1_10__created_at_update.sql

```diff
@@ -0,0 +1,2 @@
+alter table referral
```

Comment:
Related to the previous versioning PRs -- would it make sense to switch to timestamp-based versions rather than semantic versioning?

`v20210111.210044__rename_created_at.sql`

Reply:
not seen this before in MOJ but i'm into it